test_files/image/config.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
How about putting those files into the official `test` directory, like: `test/data`?